### PR TITLE
[zh] Resync docs/concepts/configuration/secret.md

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -69,603 +70,624 @@ Pod 可以用三种方式之一来使用 Secret：
...
-```
-
-<!-- The output is similar to: -->
-输出类似于：
-
-```
-MWYyZDFlMmU2N2Rm
-```
+`kubernetes.io/dockercfg` 是一种保留类型，用来存放 `~/.dockercfg` 文件的
+序列化形式。该文件是配置 Docker 命令行的一种老旧形式。
```
Comment (tanjunchen) [inline content/zh/docs/concepts/configuration/secret.md:283]: 1 存放文件

File: content/zh/docs/concepts/configuration/secret.md [hunk 2]
```diff
@@ -69,603 +70,624 @@ Pod 可以用三种方式之一来使用 Secret：
...
+The `kubernetes/dockerconfigjson` type is designed for storing a serialized
+JSON that follows the same format rules as the `~/.docker/config.json` file
+which is a new format for `~/.dockercfg`.
+When using this Secret type, the `data` field of the Secret object must
+contain a `.dockerconfigjson` key, in which the content for the
+`~/.docker/config.json` file is provided as a base64 encoded string.
+
+Below is an example for a `kubernetes.io/dockercfg` type of Secret:
 -->
+类型 `kubernetes.io/dockerconfigjson` 被设计用来保存 JSON 数据的序列化形式，
```
Comment (tanjunchen) [inline content/zh/docs/concepts/configuration/secret.md:298]: 1

File: content/zh/docs/concepts/configuration/secret.md [hunk 3]
```diff
@@ -69,603 +70,624 @@ Pod 可以用三种方式之一来使用 Secret：
...
-<!--The output is similar to: -->
-输出类似于：
+<!--
+This command creates a Secret of type `kubernetes.io/dockerconfigjson`.
+If you dump the `.dockerconfigjson` content from the `data` field, you will
+get the following JSON content which is a valid Docker configuration created
+on the fly:
+-->
+上面的命令创建一个类型为 `kubernetes.io/dockerconfigjson` 的 Secret。
+如果你对 `data` 字段中的 `.dockerconfigjson` 内容碱性转储，你会得到下面的
```
Comment (tanjunchen) [inline content/zh/docs/concepts/configuration/secret.md:352]: 碱性？

File: content/zh/docs/concepts/configuration/secret.md [hunk 4]
```diff
@@ -69,603 +70,624 @@ Pod 可以用三种方式之一来使用 Secret：
...
-is used. For example, the following Secret definition:
+The SSH authentication Secret type is provided only for user's convenience.
+You can create an `Opaque` for credentials used for SSH authentication.
+However, using the builtin Secret type helps unify the formats of your credentials
+and the API server does verify if the required keys are provided in a Secret
+configuration.
 -->
+提供 SSH 身份认证类型的 Secret 仅仅是出于用户方便性考虑。
+你也可以使用 `Opaque` 类型来保存用于 SSH 身份认证的凭据。
+不过，使用内置的 Secret 类型的有助于对凭据格式进行归一化处理，并且
```
Comment (tanjunchen) [inline content/zh/docs/concepts/configuration/secret.md:459]: 1

File: content/zh/docs/concepts/configuration/secret.md [hunk 5]
```diff
@@ -69,603 +70,624 @@ Pod 可以用三种方式之一来使用 Secret：
...
-<!--
-Results in the following secret:
+The following YAML contains an example config for a TLS Secret:
 -->
-secret 中的生成结果：
+### TLS Secret
+
+Kubernetes 提供一种内置的 `kubernetes.io/tls` Secret 类型，用来存放证书
+及其相关密钥（通常用在 TLS 场合）。
+此类数据主要提供给 Ingress 资源，用以终结 TLS 链接，不过也可以用于其他
```
Comment (tanjunchen) [inline content/zh/docs/concepts/configuration/secret.md:479]: 终结？

File: content/zh/docs/concepts/configuration/secret.md [hunk 6]
```diff
@@ -69,603 +70,624 @@ Pod 可以用三种方式之一来使用 Secret：
...
-using the `-b` option to split long lines.  Conversely Linux users *should* add
-the option `-w 0` to `base64` commands or the pipeline `base64 | tr -d '\n'` if
-`-w` option is not available.
+When creating a TLS Secret using `kubectl`, you can use the `tls` subcommand
+as shown in the following example:
 -->
-data 和 stringData 的键必须由字母数字字符 '-', '\_' 或者 '.' 组成。
+提供 TLS 类型的 Secret 仅仅是出于用户方便性考虑。
+你也可以使用 `Opaque` 类型来保存用于 TLS 服务器与/或客户端的凭据。
+不过，使用内置的 Secret 类型的有助于对凭据格式进行归一化处理，并且
```
Comment (tanjunchen) [inline content/zh/docs/concepts/configuration/secret.md:511]: 1

File: content/zh/docs/concepts/configuration/secret.md [hunk 7]
```diff
@@ -69,603 +70,624 @@ Pod 可以用三种方式之一来使用 Secret：
...
-Type:            Opaque
-
-Data
-====
-password.txt:    12 bytes
-username.txt:    5 bytes
-```
+通过将 Secret 的 `type` 设置为 `bootstrap.kubernetes.io/token` 可以创建
+启动引导令牌类型的 Secret。这种类型的 Secret 被设计用来支持节点的启动引导过程。
+其中包含用来为周知的 ConfigMap 签名的令牌。
```
Comment (tanjunchen) [inline content/zh/docs/concepts/configuration/secret.md:549]: 周知的 1

**Review by tanjunchen — COMMENTED** [review]
/approve
/lgtm

**Comment by zacharysarah** [comment]
English fixes look good. 👍 
/lgtm

**Comment by sftim** [comment]
/approve

It's unusual to fix upstream (English) and a localization at once, but it seems OK.

**Pull request merged**